game-headless/scripts/run_bot

```diff
@@ -46,21 +46,6 @@ readonly BOT_PORT=3300
 #
 readonly BOT_NAME="Bot_${USER}_${BOT_PORT}"
 
-#
-# The password used to secure access to games running on your bot.  Users
-# attempting to start a new game or connect to an existing game on your bot
-# will be prompted for this password.  If the password is an empty string,
-# the user will not be prompted, and any lobby user will be able to use your
-# bot.
-#
-# The default value is an empty password. You should change this to a
-# non-empty string if you do not want arbitrary users to use your bot.  For
-# example, you may wish to run a private game for you and some friends.  In
-# that case, set the password to a non-empty string and communicate it to
-# your friends in some manner (e.g. via email).
-#
-readonly BOT_PASSWORD=
```

Comment:
Bots are hosted for other people, password protecting bots does not really make sense. We likely do not need to pass password as a parameter for bots.

We have a lot of time before next release. IMO we need to take YAGNI pretty seriously and favor removing unused features

Reply:
There were some queries in 2018 of users wanting to host headless bots for the use of _their_ friends.  I'm pretty sure their intention was to restrict access so an arbitrary lobby user couldn't simply start a game on their private bot.

Reply:
Was that before we doubled the size of the bot fleet? 

(this is a moot consideration as I rolled back the changes in the script  files and removed just the parameters that are now hardcoded). If it were up to me we would YAGNI the :hankey: out of bots and headless and likely have something pretty different by the time we hit next release. The fact that we had required/non-required bugs, two different usage texts that could kick in is a pretty big failing. I'm a fan of not blaming people, that only leaves the blame on code complexity. Having features that are not strictly asked for or needed is not a good thing.  I appreciate the example, but even then, one example over the last *year*, I can't think of any before that, and even in the example a password is not strictly needed.

There is a connection lag on late-stage games when new players join. I'm not sure if that is fixed. Largely that is the reason for passwording, people are eager to check out the round 24 game, with a 20 second game-freeze on each new person joining, it quickly adds minutes to game-turns. IMO we should be getting it so that turns in TripleA are minutes, preferably *a* minute. That goal is why I started dev'ing for TripleA to begin with, to make it faster to play a single turn, let alone the entire game. There's been a lot to do...